discovery.md

```diff
@@ -534,9 +535,19 @@ compliant Discovery Endpoint implementations.
 
 #### `GET /services`
 
-This MUST return an array of zero or more Services. The array MUST contain all
-Services available via this Discovery Enpoint. Any Service previously returned
-to a client that does not appear in this result can be assumed to be deleted.
+This MUST return an array of zero or more Services. The array MUST contain the
+latest version of all Services available via this Discovery Enpoint. Any
+Service previously returned to a client that does not appear in this result can
+be assumed to be deleted.
```

Comment:
I wonder, if the assumption that a service is _deleted_ can be made. There might simply be a change of access rules. Technically the service in that case would still exist, it would just not be visible to me any more.

Reply:
This is an interesting point.

I unthinkingly kept this but I would expect a loss of access rights to a service to have the same result.  Since it is commentary as opposed to normative we could just remove the comment.  Yet, when I think of how I might change it I wonder if, in a practical regard, it is as if the service is deleted and the requestor should act accordingly.   The statement "... assumed to be removed from the set of Services the requestor has available" is more correct but the requestor has to decide what to do about that regardless of the reason that the service is missing from the result.  Certainly I wouldn't expect to be required to give indication of the reason as a discovery endpoint especially if I don't have any record because it was deleted.

Any suggestions on how to change this?

Reply:
maybe instead of "deleted" we say that the old version is no longer part of the result set. And whether "result set" is influence by ACL or something else is out of scope

Reply:
I will remove this out of rabbit hole avoidance.

Reply:
Actually, I think this serves a purpose.  The significance of a service being returned (or not) is that it is "on offer" or "available" for use (or not) to the requester.  Rather than "deleted" we can write "no longer available" which addresses the consequence while not making the tautological statement that not being in the result set is not being in the result set.